### ISSUE TITLE
Scary "status failed with container in error: updated to In Progress" message

### ISSUE THREAD

**Issue by tstromberg** [GoogleContainerTools/skaffold#4505]
Seen in a recent Cloud Code friction log (2020-07-15) (@steren):

```
Status check started
Resource pod/xx status failed with container install-nodejs-support in error:
Resource deployment/xy status updated to In Progress
```

Instead, it should probably say nothing.. as the deployment is in progress?

**Comment by briandealwis** [comment]
Should fix that typo too!

**Comment by dgageot** [comment]
The typo wasn't in the code :-)

**Comment by briandealwis** [comment]
I believe this was fixed with #4472.
